iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/HttpsBatchMessageTests.java

```diff
@@ -1,71 +0,0 @@
-/*
- *  Copyright (c) Microsoft. All rights reserved.
- *  Licensed under the MIT license. See LICENSE file in the project root for full license information.
- */
-
-package tests.integration.com.microsoft.azure.sdk.iot.iothub;
-
-import com.microsoft.azure.sdk.iot.device.Message;
-import com.microsoft.azure.sdk.iot.device.exceptions.IotHubSizeExceededException;
-import com.microsoft.azure.sdk.iot.device.transport.https.HttpsBatchMessage;
-import com.microsoft.azure.sdk.iot.device.transport.https.HttpsSingleMessage;
-import org.junit.Test;
-import tests.integration.com.microsoft.azure.sdk.iot.helpers.IntegrationTest;
-import tests.integration.com.microsoft.azure.sdk.iot.helpers.annotations.IotHubTest;
-
-import java.nio.charset.Charset;
-import java.nio.charset.StandardCharsets;
-
-import static org.junit.Assert.assertTrue;
-
-/** Integration tests for HttpsBatchMessage. */
-@IotHubTest
-public class HttpsBatchMessageTests extends IntegrationTest
-{
-    protected Charset UTF8 = StandardCharsets.UTF_8;
-
-    @Test
-    public void batchMessageSetsFieldsCorrectly() throws
```

Comment:
We have tests for batch messaging within a different suite that actually calls the user facing APIs rather than directly calling this internal-use-only class